src/main/webapp/script.js

```diff
@@ -6,12 +6,23 @@ function goToHub() {
   location.href = url;
 }
 
+/**
+ * Redirect to User Settings Page.
+ */
+function goToSettings() {
+  const url = 'user_settings.html';
+  location.href = url;
+}
+
 /**
  * Redirect to User Profile Page.
+ * if user is going to their own page (default)
+ * then eventually we pass their UserID
+ * @param {number} userID
  */
-function goToUser() {
+function goToUser(userID) {
```

Comment:
Perhaps change it to an optional parameter (temporarily)? Because sometimes goToUser() is called without a parameter.